runtime/driver/src/main/java/edu/snu/nemo/driver/NemoDriver.java

```diff
@@ -135,15 +139,19 @@ public void onNext(final ActiveContext activeContext) {
       final boolean finalExecutorLaunched = runtimeMaster.onExecutorLaunched(activeContext);
 
       if (finalExecutorLaunched) {
-        startSchedulingUserApplication();
+        clientRPC.send(ControlMessage.DriverToClientMessage.newBuilder()
+            .setType(ControlMessage.DriverToClientMessageType.ResourceReady).build());
       }
     }
   }
 
-  private void startSchedulingUserApplication() {
+  /**
+   * Start user application.
+   */
+  public void startSchedulingUserApplication(final String dagString) {
     // Launch user application (with a new thread)
     final ExecutorService userApplicationRunnerThread = Executors.newSingleThreadExecutor();
-    userApplicationRunnerThread.execute(userApplicationRunner);
+    userApplicationRunnerThread.execute(() -> userApplicationRunner.run(dagString));
```

Comment:
Just asking: is this the more standard way of calling a runner now?

Reply:
It's just a lambda. Since userApplicationRunner does not implement `Runnable`, we can create on-the-fly `Runnable` using lambda.